### ISSUE TITLE
When it comes to house renting in Beijing, the young people are crying

### ISSUE THREAD

**Issue by SiruiZhu** [jsoma/data-studio-projects#310]
# Pitch
Read a news about Beijing’s rental market prices are going through the roof，actually this isn't a new issue. In the first-tier cities in China, it's getting more and more expensive in terms of house renting. For graduates, it's hard for them to live in first-tier cities, even though with the good job opportunities. However, the trend is that more graduates are attracted by the rising first-tier cities(cities that are developed fast and has less pressure with good living quality) and second-tier cities. 

For this project, I simply want to see house rent in Beijing, like how much is it, and what elements effect the price most(venue, near to subway, or the area)

Another piece I read is about there are more and more house agency, they decorate the apartment, and then rent them to tenants with much high price. 

Related reading: [Financial Times: 'Beijing rent ranked world’s least affordable'](https://www.ft.com/content/bfcbd512-0789-11e6-96e5-f85cb08b0730)

## Summary

So I scraped the available house in rent on one of the biggest housing website "Lianjia", they have about 18,000 available house at the moment. I plan to use the data I scraped to do a simple analysis for this project, and saved for further using. 

Hope I can find something interesting.

## Details

**Possible headline(s):** 
Trending in Beijing's House Rent
A sketch of Beijing's House Rent

**Data set(s):** 
[House rent website: lianjia](https://bj.lianjia.com/)

**Code repository:** 
https://github.com/SiruiZhu/Data-Studio/tree/master/code/06-Bejing%20house
	
**Possible problems/fears/questions:** 
I might need to combine more dataset with this topic. 

## Work so far

I just scraped the data.

## Checklist

This checklist must be completed before you submit your draft.

- [x] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue links to my data set(s)
- [x] My issue links to my code repository
- [x] My issue explains what I'd like to explore in the data set
- [ ] My issue includes images - either inspiration or what I've done so far

**Comment by SiruiZhu** [comment]
# Update 01
	
## Your project content: images/words/etc
Image 1: We can found that Haidian district, West district and Chaoyang district have the most expensive price, however it depends on the layout of the house, whether entire house or shared rooms.
![output_4](https://user-images.githubusercontent.com/39416241/44547377-eb978780-a6e8-11e8-89d4-e4f383f31282.jpg)
Image 2: From a news, I read about that decorations by agency can also enhance the price, so here is the entire house rent decoration condition, with the price distribution:
![output1-01](https://user-images.githubusercontent.com/39416241/44547379-ec301e00-a6e8-11e8-88f8-37e966934771.png)
Here is the shared room rent decoration condition, with the price distribution:
![output2-01](https://user-images.githubusercontent.com/39416241/44547380-ec301e00-a6e8-11e8-911e-0fe95f347d2c.png)
From this graph, we can found out that area of a house has some correlation with price. 
![output3-01](https://user-images.githubusercontent.com/39416241/44547381-ec301e00-a6e8-11e8-92a9-fde912f63c51.png)

  
## Any changes in direction or topic?
Nan
  
I want to create a Choropleth map based on district with the mean of the rent price. 

## Problems/Questions
I didn't managed to scrap the house distance from the nearest subway, didn't find the infomation, but want to explore more to get this info. Compare the distance to subway from area of the room, to see which one effect on price more. 
	
## Checklist
	
- [x] I have included my visuals
- [x] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by sarahslo** [comment]
On the data side, for the decoration, I'd removed 'not stated' because the data don't tell us anything.
This is a good inquiry and I like that you are using the candlestick charting form. I don't know what the decorations styles mean though... Pudding? Miso?

**Comment by SiruiZhu** [comment]
@sarahslo Hi Sarah, thanks for your comments! :)
Yeah, will revise the 'nit stated' data. 
I just translate the decorations styles from Chinese, I think they just name their style with it, and the 3.0, and 4.0 is the class, so the higher the class, the better decoration is.

**Comment by SiruiZhu** [comment]
# Pre Final

## Project visuals/text
![output1](https://user-images.githubusercontent.com/39416241/44808713-f6e02c80-ab9a-11e8-95c1-eceff7ac0dfb.png)
![output2](https://user-images.githubusercontent.com/39416241/44808714-f6e02c80-ab9a-11e8-8e66-3a95d685e56a.png)
![output3](https://user-images.githubusercontent.com/39416241/44808715-f6e02c80-ab9a-11e8-866a-4566a9e2506a.png)
![output4](https://user-images.githubusercontent.com/39416241/44808716-f6e02c80-ab9a-11e8-944a-518345aaff0b.jpg)

## Details

**Headline:** 
A sketch of Beijing's House Rent

**Published website version:**
https://siruizhu.github.io/Beijing%20house%20rent/

**Code repository:**
https://github.com/SiruiZhu/Data-Studio/tree/master/code/06-Bejing%20house

**Final data set(s):** 
[House rent website: lianjia](https://bj.lianjia.com/)

## What did you find to be the most difficult part of this project?
1. It's hard to find a district shapefile of Beijing, so i used a mapping tool. I wanted to do a Choropleth map actually.

2. I want to get the distance of that house from nearest subway, but I didn't see the data from the website. Still need to explore more. 

## Are you satisfied with what you produced? Is there anything you would like to change or improve?
Kind of, finished the simple part. I'd definitely like to keep this dataset and this topic, can do more analysis and combine another related dataset, like income to do more stories.

## Checklist

- [x] I have included my visuals
- [x] I have posted my project to my project website
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by SiruiZhu** [comment]
# Final

## Project visuals/text
![output1](https://user-images.githubusercontent.com/39416241/44925492-9547cb80-ad1c-11e8-9ab0-23c9a70a8c0a.png)
![output2](https://user-images.githubusercontent.com/39416241/44925493-95e06200-ad1c-11e8-9c31-1db20663e589.png)
![output3--01](https://user-images.githubusercontent.com/39416241/44925498-97aa2580-ad1c-11e8-912b-b91b61025093.png)
![output3-01](https://user-images.githubusercontent.com/39416241/44925499-97aa2580-ad1c-11e8-86d3-c079c6a4c93d.png)
![output4](https://user-images.githubusercontent.com/39416241/44925502-98db5280-ad1c-11e8-9f1a-22a772d3a2ad.jpg)


## Details

**Headline:** 
A sketch of Beijing's House Rent

**Published website version:**
https://siruizhu.github.io/Beijing%20house%20rent/

**Code repository:**
https://github.com/SiruiZhu/Data-Studio/tree/master/code/06-Bejing%20house

**Final data set(s):** 
[House rent website: lianjia](https://bj.lianjia.com/)

## What did you find to be the most difficult part of this project?
1. It's hard to find a district shapefile of Beijing, so i used a mapping tool. I wanted to do a Choropleth map actually.

2. I want to get the distance of that house from nearest subway, but I didn't see the data from the website. Still need to explore more. 

## Are you satisfied with what you produced? Is there anything you would like to change or improve?
Kind of, finished the simple part. I'd definitely like to keep this dataset and this topic, can do more analysis and combine another related dataset, like income to do more stories.

## Checklist

- [x] I have included my visuals
- [x] I have posted my project to my project website
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository
